src/Dialect/ONNX/ONNXOps/OpHelper.cpp

```diff
@@ -327,18 +327,17 @@ Value createNoneFloatConstant(PatternRewriter &rewriter, Location loc) {
 }
 
 // Returns true if the Value is defined by a unit constant.
+// The unit constant can  be 1. NoneType, or 2. 1D tensor with 0 length
```

Comment:
The name `isFromNone` maybe technically correct but does not really convey the meaning of the specs. For example, CONV is defined as having an optional input  bias.

Since we are already changing names, I would recommend to use `hasOptionalInputValue(...)`, which would  returns false if "is from none" and true otherwise.

Reply:
I thought about changing the name. But the new name  containing "Optional" may be confused with the new feature OptionalType. How about leaving it?

Reply:
Yes, Optional is used in 2 context, that is why I had OptionalInput here... Maybe `IsNone`?

If you want to keep it as is, I am fine too.